File: apzkr-pzpi-21-5-kravchenko-yevhenii/Task3-WebClient/src/app/core/models/tarifs/tarif-model.ts
import { DayOfWeek } from "../../enums/day-of-week";
import { TimeUnitValue } from "../../enums/time-unit-value";

export interface TarifModel {
    id: number,
    name: string,
    activeOnDaysOfWeek: DayOfWeek[],
    startTime: Date,
    endTime: Date,
    timeUnitValue: TimeUnitValue,
    pricePerTimeUnit: number
}

File: apzkr-pzpi-21-5-kravchenko-yevhenii/Task1-Server/BLL/Infrastructure/Models/Tarif/TarifModel.cs
using Infrastructure.Enums;

namespace BLL.Infrastructure.Models.Tarif;
public class TarifModel
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public List<DayOfWeek> ActiveOnDaysOfWeek { get; set; } = new List<DayOfWeek>();

    public TimeOnly StartTime { get; set; }

    public TimeOnly EndTime { get; set; }

    public TimeUnitValue TimeUnitValue { get; set; }

    public decimal PricePerTimeUnit { get; set; }
}
